### PR TITLE
AMD GPU Fix (Again)

### PR DESCRIPTION
**Pull request by shadowbrony33**
Excuse some of the formatting edits, that was just clang mucking around.

### CODE REVIEW

File: src/xenia/gpu/xenos.h [hunk 1]
```diff
@@ -157,8 +157,10 @@ enum class ColorRenderTargetFormat : uint32_t {
 };
 
 enum class DepthRenderTargetFormat : uint32_t {
-  kD24S8 = 0,
-  kD24FS8 = 1,
+  //  kD24S8 = 0,
+  //  kD24FS8 = 1,
+  kD32S8 = 0,
```
Comment (DrChat) [inline src/xenia/gpu/xenos.h]: Don't change these names! They represent the meanings of these values to the Xenos chip, not our underlying implementations!

Comment (shadowbrony33) [inline src/xenia/gpu/xenos.h]: noted

**Comment by Gamedev1909** [comment]
Confirmed fixed amd crashes!!!

**Comment by shadowbrony33** [comment]
@Gamedev1909
Still have issues with Sonic 06 rendering, however everything else is fine.
image

@AllanCat said this issue was caused by something with k_16_16_FLOAT or k_16_16_16_16_FLOAT

**Comment by Gamedev1909** [comment]
hmm
thats my render

**Comment by shadowbrony33** [comment]
@Gamedev1909 How?

**Comment by Gamedev1909** [comment]
just works but the issue for me is well the render is messed up badly

**Comment by shadowbrony33** [comment]
@Gamedev1909 
image
This is what happens on my R9 390X

**Comment by shadowbrony33** [comment]
@Gamedev1909 What GPU and Driver are you using?

**Comment by Gamedev1909** [comment]
im using relive 18.0 for my 290x

**Comment by shadowbrony33** [comment]
@Gamedev1909 18.0? the latest is 17.7.2

**Comment by Gamedev1909** [comment]
for other cards besides vega ya but im using vega drivers

**Comment by shadowbrony33** [comment]
Wait that's the driver released for RX Vega

**Comment by Gamedev1909** [comment]
http://support.amd.com/en-us/kb-articles/Pages/Radeon-Software-Crimson-ReLive-Edition-for-Radeon-RX-Vega-Series-Release-Notes.aspx

**Comment by shadowbrony33** [comment]
I'll test this ASAP

**Comment by Gamedev1909** [comment]
also the newest change fixes nothing

**Comment by shadowbrony33** [comment]
I know, @DrChat needed that change though

**Comment by Gamedev1909** [comment]
ok cool i some what fixed PD

**Comment by shadowbrony33** [comment]
@Gamedev1909 
image
New Driver still doesn't resolve this issue...

**Comment by Gamedev1909** [comment]
Try the 3d space fix ?

**Comment by shadowbrony33** [comment]
@Gamedev1909 Please point to where that fix is.

**Comment by Gamedev1909** [comment]
https://github.com/benvanik/xenia/issues/742

**Pull request closed without merging**